.autorc

```diff
@@ -0,0 +1,86 @@
+{
+  "plugins": [
+    "git-tag",
+    "all-contributors",
+    "conventional-commits",
+    "first-time-contributor",
+    "released"
+  ],
+  "release": {
+    "prerelease": true
+  },
+  "owner": "physiopy",
+  "repo": "phys2bids",
+  "name": "Stefano Moia",
+  "email": "s.moia@bcbl.eu",
+  "labels": [
+    {
+      "name": "Majormod",
+      "changelogTitle": "💥 Breaking Change",
+      "description": "This PR breaks compatibility, and increments the major version (+1.0.0)",
+      "releaseType": "major",
+      "overwrite": true
+    },
+    {
+      "name": "Minormod",
+      "changelogTitle": "🚀 Enhancement",
+      "description": "This PR generally closes an `Enhancement` issue. It increments the minor version (0.+1.0)",
+      "releaseType": "minor",
+      "overwrite": true
+    },
+    {
+      "name": "Minormod-breaking",
```

Comment:
Is this still valid when `phys2bids` is past alpha (i.e., 1.0.0+).

Reply:
We could remove it, but honestly it' mainly there to maintain the same configuration across `physiopy`. I doubt we'll ever use that label!